src/bin/main.rs

```diff
@@ -179,10 +185,31 @@ fn main() {
         }
     };
 
-    let code_files = paths.code_files.as_ref().map(|files| {
-        PathUtil::list_all_files(&files[..])
-            .unwrap()
-            .into_iter()
+    if let Some(code_files) = &paths.code_files {
```

Comment:
It might be too dense, but you should be able to do this as a single `match`, e.g.
```rust
match (&paths.code_files, &paths.code_paths) {
  (Some(code_files), Some(code_paths)) if code_files.len() != code_paths.len() => { ... return; }
  _ => (),
}
```